big_tests/tests/privacy_SUITE.erl

```diff
@@ -29,22 +29,21 @@
 %%--------------------------------------------------------------------
 
 all() ->
-    [{group, management},
+    [
+     {group, management},
      {group, blocking},
      {group, allowing}
     ].
 
 groups() ->
-    G = [{management, [sequence], management_test_cases()},
-         {blocking, [sequence], blocking_test_cases()},
-         {my, [sequence], mytest()},
-         {allowing, [sequence], allowing_test_cases()}],
+    G = [{management, [parallel], management_test_cases()},
+         {blocking, [parallel], blocking_test_cases()},
+         {allowing, [parallel], allowing_test_cases()}],
     ct_helper:repeat_all_until_all_ok(G).
 
-mytest() ->
-    [block_jid_message_but_not_presence].
 management_test_cases() ->
-    [get_all_lists,
+    [discovering_support,
```

Comment:
Minor: could you name it `discover_support` to follow the convention (simple tense).